### PR TITLE
feat(desktop): update getting started guide to 0.3.0

### PR DESCRIPTION
**Pull request by JimmyMow**
This updates our `Zap Desktop Getting Started` tutorial for the 0.3.0-beta release

### CODE REVIEW

File: zap-desktop-getting-started.md [hunk 1]
```diff
@@ -1,92 +1,67 @@
...
 
-This tutorial assumes you have already downloaded the new Zap Desktop v0.1.2 found [here](https://github.com/LN-Zap/zap-desktop/releases).
+  This tutorial assumes you have already downloaded the new Zap Desktop v0.3.0 found [here](https://github.com/LN-Zap/zap-desktop/releases).
 
-## Step 1: Connection
-Ok let's do this dance. Once you've started Zap you will see a screen titled "How do you want to connect to the Lightning Network?"
+  - Mac users will need to download ZapDesktop-darwin-v0.3.0-beta.dmg
+  - Windows users will need to download ZapDesktop-win32-v0.3.0-beta.exe
+  - Linux users will need to download one of the following:
+      - ZapDesktop-linux-amd64-v0.3.0-beta.deb
```
Comment (mrfelton) [inline zap-desktop-getting-started.md:10]: There should be 4 files listed for linux - `.AppImage' and `.deb` each for 64 bit and 32 bit.

File: zap-desktop-getting-started.md [hunk 2]
```diff
@@ -1,92 +1,67 @@
...
-Now we will set our wallet password. This password will be used to encrypt your wallet data. Must be 8 characters in length. You'll use this password to unlock your wallet on app start up.
+  <p align='center'>
+    <img src='https://imgur.com/cRfgaL3.png' alt='screenshot' />
+  </p>
 
-<p align='center'>
-  <img src='https://i.imgur.com/Gm4SBBX.png' alt='screenshot' />
-</p>
+  ## Step 4: Wallet password
+  Now you will set your wallet password. This password will be used to encrypt your wallet data. The password must be eight characters in length. You'll use this password to unlock your wallet on start up. If you lose this password the only way to gain access to your wallet is by way of recovery via your 24 word seed. Please back up this password and keep it safe!
```
Comment (mrfelton) [inline zap-desktop-getting-started.md]: must be *at least* chars (not exactly 8)

**Review by mrfelton — CHANGES_REQUESTED** [review]

File: zap-desktop-getting-started.md (hunk 1 above)
Comment (JimmyMow) [reply zap-desktop-getting-started.md:10]: Good catch!

File: zap-desktop-getting-started.md (hunk 2 above)
Comment (JimmyMow) [reply zap-desktop-getting-started.md]: 👌

**Comment by JimmyMow** [comment]
@mrfelton updated based on feedback

**Review by mrfelton — APPROVED** [review]

**Pull request merged**